config/.stylelintrc.json

```diff
@@ -0,0 +1,91 @@
+{
+   "plugins": [
+      "stylelint-scss",
+      "stylelint-order"
+   ],
+   "rules": {
+      "scss/at-import-no-partial-leading-underscore": true,
+      "scss/at-import-partial-extension": "never",
+      "scss/at-extend-no-missing-placeholder": true,
+      "scss/at-function-pattern": "^[a-z_][a-zA-Z0-9_\\-]+$",
+      "scss/percent-placeholder-pattern": "^[a-z0-9_\\-]+$",
+      "scss/at-mixin-pattern": "^[a-z_][a-zA-Z0-9_\\-]+$",
+      "scss/dollar-variable-pattern": "^[a-z_][a-zA-Z0-9_\\-]+$",
+      "selector-id-pattern": "^[a-z0-9_\\-]+$",
+      "order/order": [
+         {
+            "type": "at-rule",
+            "name": "extend"
+         },
+         {
+            "type": "at-rule",
+            "name": "include"
+         },
+         "declarations"
+      ],
+      "at-rule-blacklist": ["debug"],
+      "selector-max-id": 0,
+      "block-closing-brace-newline-after": [
+         {
+            "ignore-at-rules": [
+               "if",
+               "else"
+            ]
+         }
+      ],
+      "block-no-empty": true,
+      "property-no-unknown": [true, {
+         "checkPrefixed": true
+      }],
+      "block-opening-brace-space-before": "always",
+      "color-hex-length": "long",
+      "color-hex-case": "lower",
+      "color-no-invalid-hex": true,
+      "declaration-bang-space-before": "always",
+      "declaration-bang-space-after": "never",
+      "declaration-block-no-duplicate-properties": true,
+      "declaration-block-trailing-semicolon": "always",
+      "declaration-colon-space-after": "always",
+      "declaration-colon-space-before": "never",
+      "no-duplicate-at-import-rules": true,
+      "declaration-no-important": true,
+      "declaration-property-value-blacklist": {
+         "border": [
+            "none"
+         ],
+         "border-top": [
+            "none"
+         ],
+         "border-right": [
+            "none"
+         ],
+         "border-bottom": [
+            "none"
+         ],
+         "border-left": [
+            "none"
+         ]
+      },
+      "property-no-vendor-prefix": true,
+      "value-no-vendor-prefix": true,
+      "selector-no-vendor-prefix": true,
+      "media-feature-name-no-vendor-prefix": true,
+      "at-rule-no-vendor-prefix": true,
+      "function-comma-space-after": "always",
+      "function-parentheses-space-inside": "never",
+      "indentation": "tab",
+      "max-nesting-depth": 3,
```

Comment:
This rule causes the sass test files to fail linting:
https://github.com/Financial-Times/origami-build-tools/blob/ee546af0e0dff8fd5e7dfcb900fa23186663ad1e/templates/component-boilerplate/test-sass-mixins.js#L8
```
 Error - Expected nesting depth to be no more than 3 (max-nesting-depth)
```